source/overview/security.rst

```diff
@@ -36,6 +36,8 @@ Transmission Security
    - Mattermost supports TLS encryption using AES-256 with 2048-bit RSA on all data transmissions between Mattermost client applications and the Mattermost server across both LAN and internet.
    - Connections to Active Directory/LDAP can be optionally secured with TLS or stunnel (E10).
    - Encryption-at-rest is available through hardware and software disk encryption solutions applied to the Mattermost database, which resides on its own server within your infrastructure. To support end user search and compliance reporting of message histories, Mattermost does not offer encryption within the database.
+   - Pre-processing methods are applied to plain text data prior to encryption, and post-processing methods applied to the cipher text data after encryption.
+   - Encryption-at-rest is available for file storage through `server-side encryption with Amazon S3-managed keys <https://docs.mattermost.com/administration/config-settings.html#enable-server-side-encryption-for-amazon-s3>`_ (E20). 
```

Comment:
We should also mention that this can be achieved with Minio in TE via disk encryption, please check with Corey?

Also, can we add to review process that IJT needs to be approver on changes to Security Overview page?